### PR TITLE
Port 50896

### PR DESCRIPTION
**Pull request by salty-blue-mango**
### What does this PR do?
Port of #50896 to master

### What issues does this PR fix or reference?
n/a

### Tests written?
No

### Commits signed with GPG?
No

### CODE REVIEW

**Comment by DmitryKuzmenko** [comment]
There are merge conflicts now. Also tests are needed for this change.

**Comment by sagetherage** [comment]
@salty-blue-mango if you can help with the merge conflicts here, pretty please?

**Comment by waynew** [comment]
@sagetherage fixed the merge conflicts, and added tests

**Review by xeacott — APPROVED** [review]

File: salt/modules/event.py [hunk 1]
```diff
@@ -108,15 +110,20 @@ def fire(data, tag):
 
         salt '*' event.fire '{"data":"my event data"}' 'tag'
     """
+    if timeout is None:
+        timeout = 60
+    timeout = timeout * 1000
     try:
-        with salt.utils.event.get_event(
-            "minion",  # was __opts__['id']
+        event = salt.utils.event.get_event(
```
Comment (Ch3LL) [inline salt/modules/event.py:117]: we should still use `with` here to ensure everything is closed properly with salt's event class.

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Comment by Ch3LL** [comment]
Just following up on this PR. Any chance you saw my review from earlier?

**Comment by Ch3LL** [comment]
ping @waynew mind fixing the merge conflict here again?

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**